.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+
+before_install:
+  - npm install -g dredd
+  - cd cucumber
+  - bundle install
+
+script:
+  - cd cucumber
+  - bundle exec cucumber
```

Comment:
The cucumber test should run the `go build` command, which in turn is what is tested.